Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/header/PageHeader.tsx

```diff
@@ -157,7 +162,7 @@ const PageHeader = () => {
                 <div className={styles.navButtons}>
                     <Link to={getAllContestsPageUrl({})} className={styles.navButton}>CONTESTS</Link>
                     <Link to="/submissions" className={styles.navButton}>SUBMISSIONS</Link>
-                    <AdministrationMenu />
+                    <AdministrationMenu buttonType={AdministrationMenuButtonType.text} isUsedInPageHeader />
```

Comment:
Why we don't check if the user can view the Administration Menu directly before rendering the component? In 99% of the cases this admin component will never be created at all, but now all it's logic is executed regardless of user role.